collector/container/Dockerfile

```diff
@@ -3,15 +3,43 @@ ARG BASE_IMAGE=ubi8/ubi-minimal
 ARG BASE_TAG=8.6
 FROM ${BASE_REGISTRY}/${BASE_IMAGE}:${BASE_TAG} AS extracted_bundle
 
-COPY bundle.tar.gz /
-COPY extract-bundle.sh /bundle/
+ARG collector_version=xxx
+ARG module_version=xxx
+ARG ROOT_DIR=.
+ENV ROOT_DIR=$ROOT_DIR
+
+COPY . /go/src/github.com/stackrox/collector
+WORKDIR /go/src/github.com/stackrox/collector
+
+RUN mkdir /bundle
+
+RUN cp "${ROOT_DIR}/bundle.tar.gz" /
+RUN cp "${ROOT_DIR}/extract-bundle.sh" /bundle/
+RUN cp "${ROOT_DIR}"/scripts/* /
+
 WORKDIR /bundle
 RUN ./extract-bundle.sh
 
+RUN echo "${collector_version}" > /COLLECTOR_VERSION
+RUN echo "${module_version}" > /bundle/kernel-modules/MODULE_VERSION.txt
+
+WORKDIR /go/src/github.com/stackrox/collector
+RUN if [[ "${collector_version}" == "xxx" ]]; then \
```

Comment:
I would expect this and the following RUN step to occur in `Dockerfile.create_bundle`. Can you move them there?

Reply:
Could you please elaborate, why would you expect them there?

Reply:
I thought it was cleaner and would make the code more straightforward, but I'm not sure how much it helps.